packages/bp/src/admin/ui/src/app/EventBus.ts

```diff
@@ -1,11 +1,11 @@
 import { auth } from 'botpress/shared'
 import { EventEmitter2 } from 'eventemitter2'
-import io from 'socket.io-client'
+import { Socket, io } from 'socket.io-client'
 
 // Copied mostly as-is from ui-studio
 class EventBus extends EventEmitter2 {
```

Comment:
@allardy Couldn't we just take the one from ui-lite?

Reply:
I'm not sure how we could reuse that specific class, ui-lite is only used with loading the webchat full screen. I think having adminSocket on ui-lite is useless, as is having guestSocket on the admin panel. This was duplicated a long time ago and we didn't check all possible combination of sockets